workflow/hosts/gaeac6.yaml

```
@@ -1,37 +1,40 @@
-BASE_GIT: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/git'
+# Paths
 DMPDIR: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/dump'
+BASE_GIT: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/git'
 BASE_DATA: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/data'
 BASE_IC: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/data/ICSDIR'
+AERO_INPUTS_DIR: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/data/GEFS_ExtData/20250310'
 PACKAGEROOT: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/nwpara'
-COMINsyn: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/com/syndat'
 HOMEDIR: '/gpfs/f6/${ACCOUNT}/scratch/${USER}'
 STMP: '/gpfs/f6/${ACCOUNT}/scratch/${USER}'
 PTMP: '/gpfs/f6/${ACCOUNT}/scratch/${USER}'
-NOSCRUB: $HOMEDIR
-ACCOUNT: drsa-precip3
+NOSCRUB: '${HOMEDIR}'
+COMINsyn: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/com/syndat'
+# BQS properties
 SCHEDULER: slurm
 QUEUE: normal
 QUEUE_SERVICE: normal
 QUEUE_DTN: 'hpss'
 PARTITION_BATCH: batch
 PARTITION_SERVICE: batch
 PARTITION_DTN: 'dtn_f5_f6'
-PARTITION_CRON: 'cron_c6'
-RESERVATION: ''
 CLUSTERS: 'c6'
 CLUSTERS_DTN: 'es'
 CONSTRAINT_DTN: 'f6'
-CHGRP_RSTPROD: 'NO'
-CHGRP_CMD: 'chgrp rstprod'
+RESERVATION: ''
+PARTITION_CRON: 'cron_c6'
+USE_SCRONTAB: 'YES'
+# HPSS properties
 HPSS_PROJECT: emc-global
-DO_ARCHCOM: 'YES'
 ARCHCOM_TO: 'hpss'
 ATARDIR: '/NCEPDEV/${HPSS_PROJECT}/1year/${USER}/${machine}/scratch/${PSLOT}'
-MAKE_NSSTBUFR: 'NO'
-MAKE_ACFTBUFR: 'NO'
+CHGRP_RSTPROD: 'NO'
+CHGRP_CMD: 'chgrp rstprod'
+# Features
+SUPPORTED_RESOLUTIONS: ['C1152', 'C768', 'C384', 'C192', 'C96', 'C48']
+DO_ARCHCOM: 'YES'
 DO_TRACKER: 'YES'
 DO_GENESIS: 'YES'
 DO_METP: 'NO'
```

I think METP can be enabled on C6.

```suggestion
DO_METP: 'YES'
```